### ISSUE TITLE
Update landmarks and labels

### ISSUE THREAD

**Issue by becka11y** [readium/readium-js-viewer#93]
The table of contents area is not labeled as a landmark. Adding this would allow screen reader users to move to the ToC using the landmark features of screen readers (list landmarks or navigate from landmark to landmark).  I would suggest labeling the ToC with a role=navigation and an aria-label="Table of Contents" or add an id to the H2 that represents the Table of Contents text and use aria-labelledby.  There is already a resource string for the "Table of Contents" text. 

Since the keyboard shortcuts refer to the Toolbar it should be labeled as such. Right now the toolbar does have a role=navigation, I would add a label of "Toolbar". This would also need an added string resource for translation. Better yet would be to add the keyboard arrow key navigation and make it  true toolbar with a role of toolbar. But, that is a stretch goal.
